woocommerce/woocommerce-admin
# Homepage: Add layout with sticky columns

Fixes https://github.com/woocommerce/woocommerce-admin/issues/4231
Replaces https://github.com/woocommerce/woocommerce-admin/pull/4234

This PR adds the layout interaction. I placed some nice pastel blocks to see how it behaves as well as a faux dismiss button.

This PR includes a fair amount of temporary code and markup which can be removed as those pieces come into play.

### Screenshots

<img width="1200" alt="Screen Shot 2020-04-28 at 3 20 19 PM" src="https://user-images.githubusercontent.com/1922453/80443726-ff869880-8963-11ea-8edf-c6b030ef351c.png">

### Detailed test instructions:

1. Scroll around the page, see if its interacting as expected.
2. Resize the screen and note the right column toggles between position sticky and static depending on window innerHeight.
3. Click "Dismiss All" to simulate clearing the inbox. The right panel will shift to the center.
